spec/controllers/application_controller_spec.rb

```diff
@@ -1,23 +0,0 @@
-require 'rails_helper'
```

Comment:
Since I have removed the switch of authenticator, there is no purpose to this test anymore